app/lib/features/chat/widgets/create_chat.dart

```diff
@@ -462,6 +418,59 @@ class _CreateChatWidgetConsumerState extends ConsumerState<_CreateChatWidget> {
       ),
     );
   }
+
+  void onUserRemove(int index) {
+    ref.read(createChatSelectedUsersProvider.notifier).update((state) {
+      final result = List<ffi.UserProfile>.from(state);
+      result.removeAt(index);
+      return result;
+    });
+  }
+
+  Future<void> onPrimaryAction(List<ffi.UserProfile> selectedUsers) async {
+    if (selectedUsers.isEmpty) {
+      widget.controller.animateToPage(
+        1,
+        duration: const Duration(milliseconds: 300),
+        curve: Curves.ease,
+      );
+      return;
+    }
+
+    if (selectedUsers.length > 1) {
+      final userIds = selectedUsers.map((u) => u.userId().toString()).toList();
+      final convo = await widget.onCreateConvo(null, null, userIds);
+      if (!context.mounted) return;
+      Navigator.of(context).pop();
+      if (convo == null) return;
+      context.pushNamed(
+        Routes.chatroom.name,
+        pathParameters: {'roomId': convo.getRoomIdStr()},
+      );
+      return;
+    }
+
+    final othersUserId = selectedUsers[0].userId().toString();
+    final client = ref.read(alwaysClientProvider);
+    String? id = checkUserDMExists(othersUserId, client);
+    if (id != null) {
+      Navigator.of(context).pop();
+      context.pushNamed(
+        Routes.chatroom.name,
+        pathParameters: {'roomId': id},
+      );
+      return;
+    }
+
+    final convo = await widget.onCreateConvo(null, null, [othersUserId]);
+    if (!context.mounted) return;
+    Navigator.of(context).pop();
+    if (convo == null) return;
+    context.pushNamed(
+      Routes.chatroom.name,
+      pathParameters: {'roomId': convo.getRoomIdStr()},
+    );
+  }
```

Comment:
let's wrap that entire part in a `try-catch` with `_log.severe` and `EasyLoading.showError()` in the catch ....

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1580/commits/2a3b0c4ae00669ce0ccba1442db910e9dc1b94ed